components/table/table-col-sort-button.js

```diff
@@ -122,8 +131,24 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 		const iconView = !this.nosort ?
 			html`<d2l-icon icon="${this.desc ? 'tier1:arrow-toggle-down' : 'tier1:arrow-toggle-up'}"></d2l-icon>` :
 			null;
+		const buttonTitle = this._getSortButtonTitle();
+		const buttonDescription = !this.nosort ? 'click to change sort order' : 'click to add sort order';
```

Comment:
All the rendered text like this needs to be run through our [`LocalizeMixin`](https://github.com/BrightspaceUI/core/tree/main/mixins/localize) so that we render in the correct language. So you need to set up the mixin, add these strings to the `en.js` lang file, then do something like this:
```suggestion
		const buttonDescription = !this.nosort ? this.localize('components.table.clickChangeSortOrder') : this.localize('components.table.clickAddSortOrder');
```